en:
  services:
    household_account_service:
      errors:
        already_has_account: 'User already has household account'
